### DISCUSSION TITLE
Most Used Languages doesn't work on my profile.

### DISCUSSION THREAD

**Discussion by ItzNxthaniel** [anuraghazra/github-readme-stats#1124] [Q&A] [answered]
Hey there,

My Most Used Languages doesn't seem to be updating properly. I have multiple commits for JavaScript & TypeScript, along with Private Java Commits but these don't need to be shown, and my "card" still just shows C#...

[My Profile](https://github.com/ItzNxthaniel)
The Card:
![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=ItzNxthaniel&bg_color=202020&title_color=cc3333&text_color=ffffff&hide_border=true&layout=compact&lang_count=10&count_private=true)

Is this working as intended or am I doing it incorrectly?

**Comment by florianbussmann** [comment, marked as answer]
The problem is that typescript is on forked repositories which are excluded currently. You can read about it here https://github.com/anuraghazra/github-readme-stats/issues/1108

If forks would be included it will look like this as github only provides the overall language for a repo and not your actual contribution

[![GitHub language stats](https://github-readme-stats-i270cdk5i-florianbussmann.vercel.app/api/top-langs/?username=ItzNxthaniel&layout=compact&langs_count=4&bg_color=202020&title_color=cc3333&text_color=ffffff&icon_color=cc3333&hide_border=true&include_forks=true)](https://github.com/anuraghazra/github-readme-stats)

Here are 2 comments about this problem
https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-855689140
https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-855693746
